FBC/FunctionalEntities/NorthAmericanEntities/USExampleIndividuals.rdf

```diff
@@ -137,13 +140,141 @@
+		<fibo-fbc-fct-breg:hasHeadquartersAddress rdf:resource="&fibo-fbc-fct-usind;TheCoca-ColaCompanyCorporateAddress"/>
+		<fibo-fbc-fct-breg:hasWebsite rdf:datatype="&xsd;anyURI">https://www.coca-colacompany.com/</fibo-fbc-fct-breg:hasWebsite>
+		<fibo-fbc-pas-fpas:hasLegalAgent rdf:resource="&fibo-fbc-fct-usjrga;CorporationTrustCompany"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;TheCoca-ColaCompanyBusinessEntityIdentifier">
+		<fibo-fbc-fct-breg:hasEntityStatus rdf:resource="&fibo-fbc-fct-breg;ActiveStatus"/>
+		<fibo-fbc-fct-breg:hasInitialRegistrationDate>1919-09-05</fibo-fbc-fct-breg:hasInitialRegistrationDate>
+		<fibo-fbc-fct-ra:isRegisteredIn rdf:resource="&fibo-fbc-fct-usjrga;DelawareBusinessEntitiesRegistry"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;TheHomeDepotBusinessEntityIdentifier">
+		<fibo-fbc-fct-breg:hasEntityStatus rdf:resource="&fibo-fbc-fct-breg;ActiveStatus"/>
+		<fibo-fbc-fct-breg:hasInitialRegistrationDate>1978-06-29</fibo-fbc-fct-breg:hasInitialRegistrationDate>
+		<fibo-fbc-fct-ra:isRegisteredIn rdf:resource="&fibo-fbc-fct-usjrga;DelawareBusinessEntitiesRegistry"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;TheHomeDepotInc-US-DE">
+		<fibo-be-le-lei:hasAddressOfLegalFormation rdf:resource="&fibo-fbc-fct-usind;TheHomeDepotIncCorporateAddress"/>
+		<fibo-fbc-fct-breg:hasHeadquartersAddress rdf:resource="&fibo-fbc-fct-usind;TheHomeDepotIncCorporateAddress"/>
+		<fibo-fbc-fct-breg:hasWebsite rdf:datatype="&xsd;anyURI">https://www.homedepot.com/</fibo-fbc-fct-breg:hasWebsite>
+		<fibo-fbc-pas-fpas:hasLegalAgent rdf:resource="&fibo-fbc-fct-usjrga;CorporationServiceCompany"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;TheProctorAndGambleCompany-US-OH">
+		<fibo-fbc-fct-breg:hasEntityStatus rdf:resource="&fibo-fbc-fct-breg;ActiveStatus"/>
+		<fibo-fbc-fct-breg:hasInitialRegistrationDate>1905-05-05</fibo-fbc-fct-breg:hasInitialRegistrationDate>
+		<fibo-fbc-fct-fse:hasDateEstablished rdf:resource="&fibo-fbc-fct-usind;TheProctorAndGambleCompanyDateEstablished"/>
+		<fibo-fbc-fct-ra:isRegisteredIn rdf:resource="&fibo-fbc-fct-usjrga;OhioBusinessFilingPortal"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;TheProctorAndGambleCompanyBusinessEntityIdentifier">
+		<fibo-fbc-fct-breg:hasHeadquartersAddress rdf:resource="&fibo-fbc-fct-usind;TheProctorAndGambleCompanyCorporateAddress"/>
+		<fibo-fbc-fct-breg:hasWebsite rdf:datatype="&xsd;anyURI">https://us.pg.com/</fibo-fbc-fct-breg:hasWebsite>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fbc-fct-usind;AlphabetIncCorporateAddress">
+		<rdf:type rdf:resource="&fibo-fbc-fct-breg;RegistrationAddress"/>
+		<rdfs:label>Alphabet Inc. headquarters  / registered address</rdfs:label>
+		<skos:definition>individual representing the registered address for Alphabet Inc.</skos:definition>
```

Comment:
Fine... but for an example, it does not seem important.  I will fix it if there are other substantive changes required.

Reply:
FYI, I am reworking ALL of the labels and definitions in this ontology to be better and consistent with one another - yet another time consuming task I had hoped to avoid, but the exxamples will be improved as a consequence.